.github/actions/netlify-deploy/action.yml

```diff
@@ -22,8 +22,8 @@ outputs:
 runs:
   using: 'docker'
   image: 'Dockerfile'
-  env:
-    NETLIFY_ACCOUNT_SLUG: ${{ inputs.NETLIFY_ACCOUNT_SLUG }}
-    NETLIFY_AUTH_TOKEN: ${{ inputs.NETLIFY_AUTH_TOKEN }}
-    NETLIFY_DIRECTORY: ${{ inputs.NETLIFY_DIRECTORY }}
-    NETLIFY_SITE_NAME: ${{ inputs.NETLIFY_SITE_NAME }}
+  args:
+    - "${{ inputs.NETLIFY_ACCOUNT_SLUG }}"
+    - "${{ inputs.NETLIFY_AUTH_TOKEN }}"
```

Comment:
These will be passed as arguments for `docker run`; are these sanitized/removed from the output? e.g. I see 

```
/usr/bin/docker run --name e87b52a6ddbf692d905bf172f9691ee9985c2b_2fbd7a --label e87b52 --workdir /github/workspace --rm -e DOCKER_BUILDKIT -e INPUT_NETLIFY_ACCOUNT_SLUG -e INPUT_NETLIFY_AUTH_TOKEN -e INPUT_NETLIFY_DIRECTORY -e INPUT_NETLIFY_SITE_NAME -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/docker.github.io/docker.github.io":"/github/workspace" e87b52:a6ddbf692d905bf172f9691ee9985c2b  "" "" "_site" "docker/docker.github.io/refactor-build"
```

Reply:
`"" ""` are the `NETLIFY_ACCOUNT_SLUG` and `NETLIFY_AUTH_TOKEN`. Github actions does hide them by default but here it seems that they are not transferred properly...

Reply:
Passing as an option is still potentially dangerous; they could be logged, etc., so if we can avoid it, we should

Reply:
By default, github actions inject the secret *only* for PR made by collaborators, so the risk is kind of handled this way.